MetaDataset.py

```diff
@@ -127,11 +121,6 @@ def __init__(self,
       self._seq_lens = None
       self._num_timesteps = None
 
-    # Will only init the needed datasets.
```

Comment:
I removed this comment as I never understood it.

Reply:
In `datasets` you could e.g. define many different datasets (e.g. `dataset1`, `dataset2`, `tmp`), but not all of them will be used. Only those are used which you use by `data_map`. The comment just says, if e.g. `tmp` is not used (via `data_map`), it will also not be initialized.

I would leave the comment in. Maybe make it more clear, if you find it confusing.

Reply:
Ah, makes sense.